File: app/src/main/java/com/imdglobalservices/psi/network/base/RequestPSIDate.java
package com.imdglobalservices.psi.network.base;

import com.imdglobalservices.psi.models.PSIDate;

/**
 * @author Dwi Setiyono <dwi.setiyono@ovo.id>
 * @since 2017.14.11
 */
public class RequestPSIDate {
    public static class Request {
        public String date;

        public Request(String date) {
            this.date = date;
        }
    }

    public static class Response extends PSIDate {
    }
}

File: app/src/main/java/com/imdglobalservices/psi/network/base/RequestPSIDate.kt
package com.imdglobalservices.psi.network.base

import com.imdglobalservices.psi.models.PSIDate

/**
 * @author Dwi Setiyono <dwi.setiyono></dwi.setiyono>@ovo.id>
 * @since 2017.14.11
 */
class RequestPSIDate {
    class Request(var date: String)

    class Response : PSIDate()
}
